Content.Shared/FingerprintReader/FingerprintReaderComponent.cs

```diff
@@ -0,0 +1,46 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.FingerprintReader;
+
+/// <summary>
+/// Component for checking if a user's fingerprint matches allowed fingerprints
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class FingerprintReaderComponent : Component
+{
+    /// <summary>
+    /// Whether the fingerprint reader is enabled
+    /// </summary>
```

Comment:
Explain how this affects behavior, like:
```suggestion
    /// <summary>
    /// Whether the fingerprint reader is enabled.
    /// If false, access attempts will always be allowed.
    /// </summary>
```